<!--
  Copyright (C) 2018-2020 LEIDOS.
  Licensed under the Apache License, Version 2.0 (the "License"); you may not
  use this file except in compliance with the License. You may obtain a copy of
  the License at
  http://www.apache.org/licenses/LICENSE-2.0
  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
  License for the specific language governing permissions and limitations under
  the License.
-->
<!--
  This file is used to launch the tactical stop and wait plugin.
-->
<launch>
    <node name="stop_and_wait_plugin" pkg="stop_and_wait_plugin" type="stop_and_wait_plugin" output="log">
      <remap from="plan_trajectory" to="plugins/StopAndWaitPlugin/plan_trajectory"/>
      <rosparam command="load" file="$(find stop_and_wait_plugin)/config/parameters.yaml"/>
    </node>
</launch>